phrosty/pipeline.py

```diff
@@ -71,6 +72,7 @@ def __init__( self, imagepath, pointing, sca ):
 
         # Intermediate files
         if self.keep_intermediate:
+            # Set to None. The path gets defined later on. 
```

Comment:
But why is it important to set them at all here?

Reply:
I understand,.  Take the sentence you just used to explain it to me and put it as a comment in the code.

Reply:
Or, just define them as None anyway, regardless of the `keep_intermediate`.